c_examples/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+cmake_minimum_required(VERSION 3.14)
+
+project(c_examples DESCRIPTION "libmicrovmi C examples" LANGUAGES C)
+
+set(CMAKE_C_STANDARD 99)
+if (WIN32)
+    set(MICROVMI_FILENAME "microvmi.dll")
+else ()
+    set(MICROVMI_FILENAME "libmicrovmi.so")
+endif ()
+
+if (WIN32)
+    # create symlink microvmi.lib -> microvmi.dll.lib in target/debug
+    # because link.exe will try to look for microvmi.lib, and cargo uses another
+    # naming it seems
+    file(CREATE_LINK
+        # original
+        microvmi.dll.lib
+        # link name
+        "${CMAKE_CURRENT_SOURCE_DIR}/../target/debug/microvmi.lib"
+        RESULT LINK_RES
+        SYMBOLIC
+    )
+endif ()
+
+
+# generate libmicrovmi.h
+add_custom_command(OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/libmicrovmi.h"
+    COMMAND cbindgen --config cbindgen.toml --crate microvmi --output "${CMAKE_CURRENT_BINARY_DIR}/libmicrovmi.h"
+    DEPENDS "${CMAKE_CURRENT_SOURCE_DIR}/../target/debug/${MICROVMI_FILENAME}"
```

Comment:
nit: It might be a good idea to switch between `target/debug` and `target/release` depending on whether you are running a cmake debug or a cmake release build respectively.

Reply:
good idea !

Reply:
okay, this is fixed